skins/laika/src/components/pages/Contact.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+	<div class="column is-full contact-form">
+		<h1 class="title">{{ $t( 'contact_form_title' ) }}</h1>
+		<span class="help is-danger has-padding-bottom-18" v-if="contactData.errors">{{ $t('contact_form_error') }}</span>
+		<form method="post" action="/contact/get-in-touch" @submit="submit" id="laika-contact">
+			<fieldset>
+				<div>
+					<label for="firstname" class="subtitle">
+						{{ $t( 'contact_form_firstname_label' ) }}
+						<span class="has-text-gray-dark">{{ $t('contact_form_optional') }}</span>
+					</label>
+					<b-field>
+						<b-input type="text"
+								id="firstname"
+								name="firstname"
+								:placeholder="$t( 'contact_form_firstname_placeholder' )"
+								v-model="formData.name.value">
+						</b-input>
+					</b-field>
+					<span v-if="formData.name.validity === Validity.INVALID" class="help is-danger">{{ $t( 'contact_form_firstname_error' ) }}</span>
```

Comment:
The logic in all the `v-if`s can be transferred into `data` or a computed property. Kinda like what we did in `Address.vue`.